projects/core/src/global-message/store/reducers/global-message.reducer.spec.ts

```diff
@@ -17,11 +17,11 @@ describe('Cart reducer', () => {
   });
```

Comment:
On line 12, we could maybe use `Action` from ngrx, instead of `any`.
On line 38 and 59 we could use `GlobalMessageState`